moveit_planners/pilz_industrial_motion_planner/CMakeLists.txt

```diff
@@ -0,0 +1,494 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(pilz_industrial_motion_planner)
+
+## Add support for C++11, supported in ROS Kinetic and newer
+add_definitions(-std=c++11)
+add_definitions(-Wall)
+add_definitions(-Wextra)
+add_definitions(-Wno-unused-parameter)
+add_definitions(-Werror)
```

Comment:
To maintain Windows compatibility, it probably make sense to only  add this GCC/Clang-specific flags under an if, see for example https://github.com/ros-planning/moveit/blob/379acf91aa3630f320db77ba85b1ca294310b0bc/moveit_core/CMakeLists.txt#L8 . 

cc @seanyen

Reply:
could we do this in a separate PR afterwards?

Reply:
> 
> 
> could we do this in a separate PR afterwards?

I guess this is up to the maintainers, I just noticed this by looking up the (great) content added.